src/renderer/components/VideoPreview/index.tsx

```diff
@@ -54,6 +54,17 @@ const VideoPreviewBase = ({ src }: Props, ref: Ref<VideoPreviewRef>) => {
         controls: {
           includeControls: [],
         },
+        keyboardShortcuts: {
```

Comment:
Probably unrelated to this PR, has the functionality for the keyboard shortcuts (removed when redux was reverted) for play, skip, etc been added back with useKeypress somewhere in the repo? If not, can you please create an issue or a ticket for it?

Reply:
Github issue here: https://github.com/chloebrett/mlvet/issues/343